docs/docs/external/getting-started/enabling-weaver-network/corda.md

```diff
@@ -34,9 +34,54 @@ Let us walk through the changes that are required in different phases of your ne
 ### Development
```

Comment:
Can you look at the "Enabling TLS" sections/items in the "Enabling Weaver in Fabric" page in my latest PR (see [docs in my fork](https://vramakrishna.github.io/weaver-dlt-interoperability/docs/external/getting-started/enabling-weaver-network/fabric/#for-data-sharing-or-asset-transfer)) for the published version), and reflect those sections in this page too (copy and adapt)?

Reply:
Also, reorganize sections the same way I did in the Fabric page. I moved the phase sections one level up, so we can call out the interoperability modes as subsections rather than bullet items. It makes the document more readable.

Reply:
Done